apps/nextjs-website/src/components/Header.tsx

```diff
@@ -3,28 +3,53 @@ import { Header as HeaderEC } from '@pagopa/pagopa-editorial-components/dist/com
 import { HeaderProps } from '@pagopa/pagopa-editorial-components/dist/components/Header/Header';
 import { Stack } from '@mui/material';
 import { usePathname } from 'next/navigation';
+import { theme } from '../app/theme';
 import Icon from './Icon';
 import { HeaderWithNavigation } from '@/lib/header';
 
+declare module '@mui/material/styles' {
+  interface Theme {
+    custom: {
+      boxShadow: string;
+    };
+  }
+  // allow configuration using `createTheme`
+  interface ThemeOptions {
+    custom?: {
+      boxShadow?: string;
+    };
+  }
+}
+
```

Comment:
why is this needed?

Reply:
without that part, this line here

`boxShadow: theme.custom.boxShadow,`

will have this error here: Property 'custom' does not exist on type 'Theme'

due to not use hardcoded value inside file, I proceeded to create the theme.ts file to use it for custom colour value and I didn't find another way to do this. I'm not a fan of declaring that part inside header.tsx but I didn't found another way

Reply:
You should be able to use theme-aware values.

I'm making a quick suggestion to show the changes.